protecode/scanning_util.py

```diff
@@ -368,7 +378,7 @@ def enumerate_reference_triages():
                             f'{resource.access.imageReference=} no longer available to protecode '
                             f'- will upload. Corresponding product: {protecode_app.product_id()}'
                         )
-                        images_to_upload.append(resource)
+                        elements_to_upload.append(upload_element)
```

Comment:
naming-nit: the term we use in cnudie would be either `resources`, or - covering also `sources` - `artefacts`. Please choose either `artefact` or `resource`, if doing a rename.

note that currently, the artefacts will always be `OciImages`, as we did so far not implement this for any other artefact types.